python/TestHarness/schedulers/Job.py

```diff
@@ -350,8 +350,8 @@ def getJointStatus(self):
         Return the most accurate message possible, based on the statuses of both the Tester and Job.
         Most of the time, we want to return a Tester status. As this class will have the most detail.
         """
-        # Tester has no status, use a job status instead
-        if self.isTimeout() or self.isNoStatus():
+        # Job has failed, or tester has no status
+        if self.isError() or self.isNoStatus():
```

Comment:
This change seems unrelated. This is intentional correct?

Reply:
That was concerning the comment I made in commit 6bb2201:

Fix an interesting issue discovered during this process.
- A skipped test, which has failed due to dependencies was not
being detected properly.